core/eval.go

```diff
@@ -815,6 +815,49 @@ func evalSTACKREFPEEK(args []string) []byte {
 	return Encode(s.Iterate(int(num)), false)
 }
 
+// evalWATCH adds the specified key to the watch list for the caller client.
+// Every time a key in the watch list is modified, the client will be sent a response
+// containing the new value of the key along with the operation that was performed on it.
+// Contains only one argument, the key to be watched.
+func evalWATCH(args []string, c *Client) []byte {
```

Comment:
Let's rename WATCH to QWATCH.

Rationale: Redis already has the command WATCH. Today DiceDB does not implement it, but we can become a drop-in replacement of Redis (if required). Hence I propose adding a new command called QWATCH.

ps: QWATCH stands of Query WATCH.

Reply:
Done.